docs/UserGuide.md

```diff
@@ -100,12 +114,42 @@ Format: `DURATION_AMOUNT DURATION_UNIT`
 
 ### 4.1. Accessing help: `help`
 
-Shows you help text for the command word supplied. If no command word is supplied, you will be shown a generic help message instead.
-
-Format: `help [COMMAND_WORD]`
-
-- `COMMAND_WORD`: The command you wish to view the help message for.
-
+- Shows you a generic help message instead.
```

Comment:
The help and list command are similar in the sense that they both have an optional parameter in the command, but the format for both commands are abit different. Maybe the help command can be rephrased as 
`If no COMMAND_WORD is given, a generic help message is displayed instead.`